.github/workflows/dotnet-build.yml

```diff
@@ -85,6 +90,8 @@ jobs:
   build_library:
     name: Build Library
     runs-on: ${{ matrix.os }}
+    env:
+      VSTEST_TESTHOST_SHUTDOWN_TIMEOUT: 1100 # Fixes "The active test run was aborted. Reason: Test host process crashed" 
```

Comment:
Do we have this issue on new pipeline?
I also curious if this variable is specific to Azure Pipelines or exists in Github.

Reply:
Haha - I wish I knew the answer to that! 

I don't have direct evidence that supports whether we still need it in GitHub Actions or not, but I'd kinda rather keep it just in case since it doesn't introduce any adverse effects to the pipeline and it may save us headaches in the future.